### PR TITLE
Feat: add artist credit for team illustrations

### LINKED ISSUE #125: Credit artist for the employee illustrations on About page
Please credit @worksofharte (see her Instagram url below) for the illustrations on the About page. Perhaps something like "Illustrations by @worksofharte" underneath the illustrations?
https://www.instagram.com/worksofharte/

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #125

### CODE REVIEW

File: styles/base.css [hunk 1]
```diff
@@ -278,6 +279,18 @@ header {
     }
 }
 
+/* Credit */
+.credit {
+    color: var(--text-grey);
+    font-family: var(--bs-font-monospace);
+    font-size: 12px;
```
Comment (thekaveman) [inline styles/base.css:286]: We use `rem` everywhere else `font-size` is declared, should probably do so here as well?

Comment (angela-tran) [reply styles/base.css:286]: Thanks for catching 🙏 I grabbed this CSS from Figma and thought I had it all cleaned up - missed this font unit

**Comment by segacy1** [comment]
@angela-tran could you make the weight of "worksofharte" the same as "team illustrations by"? It should be bolder

**Comment by thekaveman** [comment]
> @angela-tran could you make the weight of "worksofharte" the same as "team illustrations by"? It should be bolded

This may be because the font-weight is specified as `800` but we only bring in `500-700`: https://github.com/compilerla/compiler.la/blob/main/_layouts/default.html#L31

**Comment by angela-tran** [comment]
I think it's because the font weight from `a` element styles ends up overriding 😳 

image

I'll fix it so the link is just as bold as the text

**Comment by machikoyasuda** [comment]
Finishing up this PR by adding proper font-weight to the link. Refactored CSS.

Ready for review @angela-tran @thekaveman

**Review by thekaveman — APPROVED** [review]

**Pull request merged**